Data Set: Phylogeny of Gonioctena leaf beetles (Coleoptera; Chrysomelidae):

In order to explore the evolutionary journey of leaf beetles in adapting to new host plants, a thorough investigation was carried out. Using allozyme data, a phylogenetic analysis of the genus Gonioctena (Coleoptera, Chrysomelidae) was conducted. By examining various DNA sequences and employing phylogenetic and maximum likelihood trees, valuable insights were obtained regarding the genus's host-plant switches. To delve deeper into this phenomenon, I meticulously aligned the DNA sequences of 20 distinct species within the genus, including G. viminalis (1), G. Olivacea (1), G. rufipes, G. linnaeana, G. variabilis (1), G. fornicata, G. occidentalis, G. interposita (l), G.interposita (2), G. intermedia, G. quinquepunctata (2), G. holdausi, G. pallida (2), G. pallida (3), G. tredecimmaculata, G. nigropiagiata, G. kamikawai, G. rubripennis, Oreina cacaiiae, chrysomela tremula. These aligned sequences helped construct phylogenetic trees, which visually represent the relationships between the species. Through analyzing these distance and parsimony trees, as well as conducting extensive research, compelling evidence emerged: the Fabaceae family serves as the host-plant family for Gonioctena. Furthermore, the investigation highlighted multiple instances of host-plant shifts within the genus. My data set can be found in this journal: https://pubmed.ncbi.nlm.nih.gov/11975339/

Multiple Sequence Alignments:
Installing programs: ClustalW and MAFFT

ClustalW:
I downloaded miniconda (https://docs.conda.io/en/latest/miniconda.html), and used it to install ClustalW. I did this by i=using the code "conda install -c bioconda clustalw.‚Äù


MAFFT:
Additionally, I downloaded MAFFT (https://mafft.cbrc.jp/alignment/software/).

Both software programs are able to run in the terminal by just typing the name in the command line.  Before launching the program, ensure that you navigate to the directory containing the desired data by using the "cd" command.

Formatting Data for Alignment: 
To format my data for alignment, I had to copy and paste the sequences for each gene marker into a Visual Studio Code-3. I saved each one as a text file, in botany-563-final in data_clean as COII.txt, 12S.txt, and 16S.txt. All data files should be saved as a .txt as I did properly. 

Aligning: 
